README.md

```diff
@@ -3,26 +3,29 @@
 
 # Neomake
 
-Neomake is a plugin for asynchronous `:make` using
-[Neovim's](http://neovim.org/) job-control functionality. It is inspired by
-the excellent Vim plugins [Syntastic](https://github.com/scrooloose/syntastic)
-and [dispatch.vim](https://github.com/tpope/vim-dispatch).
+Neomake is a plugin that asynchronously runs programs using
+[Neovim]'s job-control functionality. It is intended to
+replace Vim's built-in `:make` command and provides functionality similar to
+plugins like [syntastic] and [dispatch.vim]. It is primarily used to run code
+linters and compilers from within Vim, but can be used to run any program.
 
 **This plugin also works in ordinary Vim, but without the asynchronous
 benefits.**
 
-The minimum Neovim version supported by Neomake is `NVIM
-0.0.0-alpha+201503292107` (commit `960b9108c`).
-The minimum Vim version supported by Neomake is 7.4.503 (although if you don't
-use `g:neomake_logfile` older versions will probably work fine as well).
+## Requirements
+
+The minimum [Neovim] version supported by Neomake is
+`NVIM 0.0.0-alpha+201503292107` ([commit 960b9108c]). The minimum Vim version
+supported by Neomake is 7.4.503 (although if you don't use `g:neomake_logfile`
+older versions will probably work fine as well).
 
 ## How to use (basic)
 
 Just set your `makeprg` and `errorformat` as normal, and run:
 
     :Neomake!
 
-If your makeprg can take a filename as an input, then you can run `:Neomake`
+If your `makeprg` can take a filename as an input, then you can run `:Neomake`
```

Comment:
This is also what https://github.com/neomake/neomake/issues/589 is about, isn't it?